app/views/docs/getting-started-for-server.phtml

```diff
@@ -415,10 +417,13 @@ $user = $users->create(
     <li>
         <h3>Python</h3>
         <div class="ide margin-top-small" data-lang="python" data-lang-label="Python SDK">
-            <pre class="line-numbers"><code class="prism language-python" data-prism>users = Users(client)
+            <pre class="line-numbers"><code class="prism language-python" data-prism>from appwrite.id import ID
```

Comment:
This example continues from previous examples and doesn't show the imports for brevity and continuity. All the other examples also don't show imports.

If you believe this section without imports seems confusing, consider how you would rephrase the sentences above so readers understand that this is the next step from previous examples.

Otherwise, consider reverting this change :)

Reply:
So for me it was pretty confusing following this tutorial, because I didn't know where to import these files from.

Reply:
So for me it was pretty confusing following this tutorial, because I didn't know where to import these files from.

Reply:
> So for me it was pretty confusing following this tutorial, because I didn't know where to import these files from.

I recommend looking at the sections above this to see if there is a more natural way to lead the user to this section, if not, please update all examples in the docs here for consistency. 

Thanks ❤️

Reply:
I added a note.  
Maybe this is too much - if so, I think it's no problem to revert this change.  
Maybe I was the only stupid one to don't look down 4 rows to see the complete example :)